docs/source/notebooks/mmm/mmm_example.ipynb

```diff
@@ -10302,7 +10302,7 @@
    "metadata": {},
```

Comment:
<p>There is duplicated word "see See"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='141'/>

Reply:
good catch!